### PR TITLE
Feat/bifold

### PR DESCRIPTION
**Pull request by cre8**
Signed-off-by: Mirko Mollik <mirko.mollik@fit.fraunhofer.de>

### CODE REVIEW

File: wallets/bifold.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "capability": ["holder"],
```
Comment (cvarjao) [inline wallets/bifold.json:2]: The `verifier` capability is in preview, but it does exists

**Review by cvarjao — COMMENTED** [review]
I think we should add `"credentialFormats": ["AnonCred", "SD-JWT-VC"]`.

Thanks you so much for adding this!

**Comment by cre8** [comment]
> I think we should add `"credentialFormats": ["AnonCred", "SD-JWT-VC"]`.
> 
> Thanks you so much for adding this!

Is bifold also supporting other signature algorithms or key managements like `did:jwk` for the holder that is not in the profile of Anoncreds with did:indy? https://openwallet-foundation.github.io/credential-format-comparison-sig/#/profiles/AnonCreds%20with%20did:indy

**Comment by cre8** [comment]
In one of the calls we said that we will not add white label wallets that are not published as a final solution.

However we will add a new column that will allow wallets to point to special requirements like important libraries projects. So speaking of bifold, the BC Wallet would reference the project as one of the dependencies.

**Pull request closed without merging**